test/Conversion/ExportVerilog/prepare-for-emission.mlir

```diff
@@ -146,6 +146,26 @@ module attributes {circt.loweringOptions = "disallowExpressionInliningInPorts"}
  }
 }
 
+// -----
+
+module attributes {circt.loweringOptions = "disallowExpressionInliningInPorts"} {
+  // CHECK-LABEL: @DisableIff
+  hw.module @DisableIff(in %clk: i1, in %a: i1, in %b: i1) {
+    %b_xor_b = comb.xor %b, %b : i1
+  
+    // CHECK: %[[XOR:.+]] = comb.xor
+    // CHECK: %[[WIRE:.+]] = sv.wire
+    // CHECK: sv.assign %[[WIRE]], %[[XOR]]
+    // CHECK: %[[READ:.+]] = sv.read_inout %[[WIRE]]
+    // CHECK: ltl.disable %{{.+}} if %[[READ]]
+    %i0 = ltl.implication %a, %b : i1, i1
+    %k0 = ltl.clock %i0, posedge %clk : !ltl.property
+    %k5 = ltl.disable %k0 if %b_xor_b : !ltl.property
```

Comment:
Seeing the code above, I'm starting to wonder if the `ltl.clock` op also needs this special treatment where the x in `posedge x` must be a wire, not an inlined expression. Should we add `ltl.clock` to the list alongside `AlwaysOp` and `AlwaysFFOp`, to ensure we don't create `posedge (x + 42)`?

Reply:
On it, will push shortly.

Reply:
Updated, thanks! Hopefully this is easy to extend if anything else is missed or added :+1: .